anvil-python/anvil/commands/haproxy.py

```diff
@@ -183,21 +209,16 @@ def extra_tfvars(self) -> dict[str, Any]:
             self.client, self._HAPROXY_CONFIG
         )
 
-        cert_filepath = variables["ssl_cert"]
-        key_filepath = variables["ssl_key"]
-        if cert_filepath != "" and key_filepath != "":
-            with open(cert_filepath) as cert_file:
-                variables["ssl_cert_content"] = cert_file.read()
```

Comment:
we want the file content to pass only to terraform and keep the user's answer on the answer dict. Why are those lines removed and the lines above added?

Reply:
In addition, This is the place to check that both of them are set and if not fail. The code that is calling this function will exit if TerraformException is raised. You can raise it yourself:

```py
        except TerraformException as e:
            return Result(ResultType.FAILED, str(e))
```

https://github.com/canonical/snap-openstack/blob/anvil-0.1/sunbeam-python/sunbeam/jobs/steps.py#L97-L112